docs/core/runtime-config/garbage-collector.md

```diff
@@ -669,6 +669,7 @@ Project file:
 - Specifies the threshold size, in bytes, that causes objects to go on the large object heap (LOH).
 - The default threshold is 85,000 bytes.
 - The value you specify must be larger than the default threshold.
+- The value can be capped by the runtime to the maximum possible size for the current configuration.
```

Comment:
Should we add that this is in .NET 9 and later, or is that unnecessary?

Reply:
I think it is unnecessary. In principle, the value should have been capped in .net 7/8 as well, we just don't have a business reason to fix those earlier versions, yet. We might if we find a reason to.